src/pages/example-1/Example1.tsx

```diff
@@ -1,25 +1,61 @@
-import React from 'react';
+import { useState } from 'react';
 import './Example1.css';
 import Table from '../../components/table/Table';
 import { useNavigate } from 'react-router-dom';
 import { AppRoutes } from '../../routes/routes';
 import { useCertificates } from '../../hooks/useCertificates';
+import ActionMenu from '../../components/action-menu/ActionMenu';
+import GearIcon from '../../icons/GearIcon';
+import ConfirmationModal from '../../components/confirmation-modal/ConfirmationModal';
 
 const Example1: React.FC = () => {
-  const headers = ['Supplier', 'Certificate type', 'Valid from', 'Valid to'];
   const navigate = useNavigate();
   const { certificates, deleteCertificate } = useCertificates();
+  const [activeMenu, setActiveMenu] = useState<number | null>(null);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [deleteIndex, setDeleteIndex] = useState<number | null>(null);
+
+  const columns = [
+    { header: 'Supplier', accessor: (row: any) => row.supplier },
```

Comment:
please do not use 'any'. We should always avoid using any, there is no point using typescript if we would use any. You should use Certificate instead of any